.github/workflows/python-publish.yml

```diff
@@ -23,10 +22,31 @@ jobs:
         run: |
           python -m pip install --upgrade pip
           pip install setuptools wheel twine
-      - name: Build and publish
-        env:
-          TWINE_USERNAME: ${{ secrets.PYPI_USERNAME }}
-          TWINE_PASSWORD: ${{ secrets.PYPI_PASSWORD }}
+      - name: Build
         run: |
           python setup.py sdist bdist_wheel
-          twine upload dist/*
+      - name: Upload artifacts
+        uses: actions/upload-artifact@v3
+        with:
+          path: dist/*
+          name: artifacts
+  Publish:
+    needs: [build]
```

Comment:
I assume this name isn't required to match exactly, as in `[Build]`?

Reply:
I guess not as it ran perfectly fine without it matching